Document improved Pages multiworker experience (#19175)
cloudflare/cloudflare-docs: src/content/docs/pages/functions/bindings.mdx

For example, to develop locally, if your Worker is called `my-worker`, run `npx wrangler dev` in the `my-worker` directory. In a different terminal, also run `npx wrangler pages dev <OUTPUT_DIR> --service MY_SERVICE=my-worker` in your Pages project directory. Interact with this Service binding by using `context.env` (for example, `context.env.MY_SERVICE`).

> Wrangler also supports running your Pages project and bound Workers at once with one command. To try it out, pass multiple -c flags to Wrangler, like this: `wrangler dev -c wrangler.toml -c ../other-worker/wrangler.toml`. The first config must point to your Pages configuration file,and the remaining configs will be accessible via a service binding from your Pages project.

Reviewer: ```suggestion
Wrangler also supports running your Pages project and bound Workers at once with one command. To try it out, pass multiple -c flags to Wrangler, like this: `wrangler dev -c wrangler.toml -c ../other-worker/wrangler.toml`. The first argument must point to your Pages configuration file, and the subsequent configurations will be accessible via a Service binding from your Pages project.
```

Reviewer 2: +1

Reviewer 3: +2

> Wrangler also supports running your Pages project and bound Workers at once with one command. To try it out, pass multiple -c flags to Wrangler, like this: `wrangler dev -c wrangler.toml -c ../other-worker/wrangler.toml`. The first config must point to your Pages configuration file,and the remaining configs will be accessible via a service binding from your Pages project.

Reviewer 2: shouldn't this be running `wrangler pages dev -c ...`?

> Wrangler also supports running your Pages project and bound Workers at once with one command. To try it out, pass multiple -c flags to Wrangler, like this: `wrangler dev -c wrangler.toml -c ../other-worker/wrangler.toml`. The first argument must point to your Pages configuration file, and the subsequent configurations will be accessible via a Service binding from your Pages project.

Author: ```suggestion
Wrangler also supports running your Pages project and bound Workers in the same dev session with one command. To try it out, pass multiple -c flags to Wrangler, like this: `wrangler pages dev -c wrangler.toml -c ../other-worker/wrangler.toml`. The first argument must point to your Pages configuration file, and the subsequent configurations will be accessible via a Service binding from your Pages project.
```

:::caution

> Support for running multiple Workers at once with one Wrangler command is experimental, and subject to change as we work on the experience. If you run into bugs or have any feedback, [open an issue on the workers-sdk repository](https://github.com/cloudflare/workers-sdk/issues/new)

Reviewer 2: I wonder if there is a better phrasing than "at once". That leaves room for interpretation ...a bit ....IMHO.  Would "in the same dev sessions" be better do you think?

> Support for running multiple Workers at once with one Wrangler command is experimental, and subject to change as we work on the experience. If you run into bugs or have any feedback, [open an issue on the workers-sdk repository](https://github.com/cloudflare/workers-sdk/issues/new)

Author: ```suggestion
Support for running multiple Workers in the same dev session with one Wrangler command is experimental, and subject to change as we work on the experience. If you run into bugs or have any feedback, [open an issue on the workers-sdk repository](https://github.com/cloudflare/workers-sdk/issues/new)
```
